sky/shell/platform/ios/sky_surface.mm

```diff
@@ -179,6 +185,13 @@ - (void)keyboardWillBeHidden:(NSNotification*)notification {
   _engine->OnViewportMetricsChanged(_viewportMetrics.Clone());
 }
 
+- (void)onLocaleUpdated:(NSNotification*)notification {
+    NSLocale *currentLocale = [NSLocale currentLocale];
```

Comment:
Yeah, it's Foo*, but I'm not gonna send out an update since this is all refactored in https://github.com/flutter/engine/pull/2471 anyway.